snippets/text/moonriver-launch/post-launch.md

```diff
@@ -0,0 +1 @@
+The Moonriver network has completed its launch process, and the staking feature is open to users that wish to participate. 
```

Comment:
I liked it before because I thought it had relevant information like staking dapp instructions, and I see why you removed that part, but without it it seems kinda unnecessary